plugins/poll/plugin.rb

```diff
@@ -467,6 +467,7 @@ def groupable_user_fields
 
       after_save do
         polls = self.extracted_polls
+        self.extracted_polls = []
```

Comment:
@ZogStriP are you happy with `after_save` doing the reset here?

Reply:
Yes, I think that's fine.

```suggestion
        self.extracted_polls = nil
```

`nil` or `{}` would be slightly more correct.